src/rust/bin/inspect/keys.rs

```diff
@@ -0,0 +1,167 @@
+                    bech32::Variant::Bech32,
+                )
+                .unwrap(),
+            );
+
+            let transparent_fvk = match AccountPrivKey::from_seed(&network, &seed, account)
+                .map(|sk| sk.to_account_pubkey())
+            {
+                Ok(fvk) => {
+                    eprintln!("   - Transparent:");
+                    match fvk.derive_external_ivk().map(|ivk| ivk.default_address().0) {
+                        Ok(addr) => eprintln!(
+                            "     - Default address: {}",
+                            match addr {
+                                TransparentAddress::PublicKey(data) => ZcashAddress::from_transparent_p2pkh(addr_net, data),
+                                TransparentAddress::Script(_) => unreachable!(),
+                            }.encode(),
+                        ),
+                        Err(e) => eprintln!(
+                            "    ⚠️  No valid transparent default address for this account under this seed: {:?}",
+                            e
+                        ),
+                    }
+
+                    Some(fvk)
+                }
+                Err(e) => {
+                    eprintln!(
+                        "  ⚠️  No valid transparent key for this account under this seed: {:?}",
+                        e
+                    );
+                    None
+                }
+            };
+
+            let items: Vec<_> = iter::empty()
+                .chain(
+                    orchard_fvk
+                        .map(|fvk| fvk.to_bytes())
+                        .map(unified::Fvk::Orchard),
+                )
+                .chain(Some(unified::Fvk::Sapling(
+                    sapling_extfvk_bytes[41..].try_into().unwrap(),
+                )))
+                .chain(
+                    transparent_fvk
+                        .map(|fvk| fvk.serialize()[..].try_into().unwrap())
+                        .map(unified::Fvk::P2pkh),
+                )
+                .collect();
+            let item_names: Vec<_> = items
+                .iter()
+                .map(|item| match item {
+                    unified::Fvk::Orchard(_) => "Orchard",
+                    unified::Fvk::Sapling(_) => "Sapling",
+                    unified::Fvk::P2pkh(_) => "Transparent",
+                    unified::Fvk::Unknown { .. } => unreachable!(),
+                })
+                .collect();
+
+            eprintln!("   - Unified ({}):", item_names.join(", "));
+            let ufvk = unified::Ufvk::try_from_items(items).unwrap();
+            eprintln!("     - UFVK: {}", ufvk.encode(&addr_net));
+        }
+        seed.zeroize();
+    } else {
+        eprintln!("🔎 To show account details, add \"network\" (either \"main\" or \"test\") and \"accounts\" array to context");
+    }
+
+    eprintln!();
+    eprintln!(
+        "WARNING: This mnemonic phrase is now likely cached in your terminal's history buffer."
```

Comment:
Please file an issue to improve this, e.g. by requiring the phrase to be entered on the terminal and then clearing the terminal window, similar to `zcashd-wallet-tool`.

Reply:
#6126